plugins/modules/homebrew.py

```diff
@@ -76,6 +76,12 @@
         type: list
         elements: str
         version_added: '0.2.0'
+    force_formula:
+        description:
+            - Force the package(s) to be treated as a formula (equivalent to C(brew --formula)).
```

Comment:
How about mentioning the homebrew_cask module here for users that are interested in casks:
```suggestion
            - Force the package(s) to be treated as a formula (equivalent to C(brew --formula)).
            - To install a cask, use the M(community.general.homebrew_cask) module.
```
Otherwise I can think of users that start thinking "ok, this adds `--formula`, but how do I tell the module to add `--cask` to the command line?".

Reply:
(Forgot to add: besides this, I think the PR looks good :) )

Reply:
Oh, good call! It's this kind of thinking I'd like to get better at.